optimism/src/keccak/mod.rs

```diff
@@ -10,6 +10,7 @@ use kimchi::circuits::polynomials::keccak::constants::{
 pub mod column;
 pub mod constraints;
 pub mod environment;
+#[cfg(feature = "bn254")]
```

Comment:
Can be removed.

Reply:
But if I remove it then it does not find `checker` in `folding`.

Reply:
Fixed in 2182